workflow/rules/datavzrd.smk

```diff
@@ -1,6 +1,8 @@
 rule split_call_tables:
     input:
-        "results/tables/{group}.{event}.variants.fdr-controlled.tsv",
+        calls="results/tables/{group}.{event}.variants.fdr-controlled.tsv",
+        population_db=get_population_db(use_before_update=True),
```

Comment:
In case of reruns the current sample will be part of the db. Do we ignore the case or should we perform a cleanup rule first?